Request/ParamFetcher.php

```diff
@@ -213,7 +213,7 @@ public function cleanParamWithRequirements(Param $config, $param, $strict)
                 return $default;
             }
             $constraint = new Regex(array(
-                'pattern' => '#^'.$config->requirements.'$#xsu',
+                'pattern' => '#^('.$config->requirements.')$#xsu',
```

Comment:
please use a non-capturing group (adding a new capturing group makes it slower for all cases, and can break cases using advanced regex features)